docs/data/sources/client-side-vs-server-side.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Client-side vs Server-side Sources
+description: Learn the difference between Amplitude's client-side and server-side sources.
+---
+
+Client-side and server-side are terms that describe where an app's code runs: either on the user's device (client-side), or on a server (server-side). Amplitude has several types of sources to cover each of your needs. This doc primarily describes the differences between client-side and server-side sources, and gives a brief overview of third-party sources. 
+
+## Client-side sources
+
+Use client-side sources in apps that your users run on their own devices, like mobile, web browser, and desktop apps. In these types of sources, code runs on the user's device.
+
+Amplitude's client-side sources include these SDKs:
+
+- Web: Browser, Marketing Analytics Browser, React Native
+- Mobile: Android, iOS, Unity Plugin, Flutter, React Native
+  
+See the [Data client-side SDKs](/data/sdks/#data-client-side-sdks) and the [Experiment client-side SDKs](/data/sdks/#experiment-client-side-sdks)
+
+## Server-side sources
+
+Use server-side sources in secure, multi-user environments like web servers and services that you run on your own servers. In these types of sources, code runs on the server. 
+
+Amplitude's server-side sources include these SDKs and APIs:
+
+- Node.js SDK
+- Go SDK
+- Python SDK
+- Java SDK
+- HTTP V2 API 
```

Comment:
I think the client-side SDKs are using these ingestion API endpoints too. Not sure whether we put these endpoints here. Example: https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-core/src/constants.ts#L4-L7.

Reply:
@liuyang1520 this doc is suggestions for the reader to implement, not necessarily explaining how our client-side SDKs operate on a technical level. If our end-users don't use HTTP V2 and Batch APIs server-side on their own, I can take them out. LMK what you think!

Reply:
Thanks @caseyamp for the explanation! Sorry for the confusion, I actually mean that the V2 API and Batch API endpoints are for both client and server side SDKs as well as reader's own implementations. For example, they can implement their own client events sender with these endpoints in the web/mobile without using our SDKs. I think server-side might use these more often (as there are some languages like php, ruby we don't cover), so feel free to leave it here.